### DISCUSSION TITLE
Add 'es' language

### DISCUSSION THREAD

**Discussion by diegort** [totaldebug/atomic-calendar-revive#1474] [Feature Requests]
Can you please add Spanish as one of the available languages.

Thanks

**Comment by marksie1988** [comment]
@diegort Thanks for the PR, I have merged it, please test the beta version that has just released and let me know if it works as expected :)

**Comment by diegort** [comment]
Hello Steven,
I've downloaded last version using HACS and I've ensured it is the proper
one but still I see text in English. However, warning in console has
disappeared, so it seems the locale has been identified...
Any clue?
-----
Diego Montesinos Hervás

**Comment by diegort** [comment]
Hello Steven,
Yesterday I found the bug, which was that I mapped incorrectly the locale
code to the translations file. 'en-GB' locale was mapped to es.json and
'es' locale was mapped to en.json. Copy/Paste problems everywhere 😅
The PR is updated, or better said, it is a new one including final status
which should work now

Best regards
-----
Diego Montesinos Hervás
